src/command/output.rs

```diff
@@ -18,6 +18,7 @@ use crate::utils::table::{self, cell, row};
 #[derive(Clone, PartialEq, Debug)]
 pub enum RoverStdout {
     DocsList(HashMap<&'static str, &'static str>),
+    SupergraphSdl(String),
```

Comment:
Definitely a nit, feel free to disregard 😄 I wonder if we'd be fine using the SDL variant here? 

Or better yet, I know we have a lot of variants that just print a string with a label above it. Maybe we could instead use a `LabeledString(String, String)` variant or something to make everything consistent and to keep us from having to `print_descriptor` and `print_content` over and over again.

If we ever did something like syntax highlighting SDL in terminal outputs, I could imagine having more specific variants like `SDL` be useful, but I'm not seeing much usefulness out of breaking them up into multiple variants rihgt now

Reply:
mmmm, i'd like to keep this as is for now since they do have a separate descriptor - can we handle a refactor like this in a separate issue/PR combo?

Reply:
Totally, just a passing thought!